ang/crmCaseType.css

```diff
@@ -32,6 +38,28 @@
     width: 10em;
 }
 
+.crmCaseType-editable-check {
+    bottom: -24px;
+    cursor: pointer;
+    height: 24px;
+    left: 0;
+    position: absolute;
+    text-align: center;
+    width: 30px;
+    z-index: 1;
+}
+
+.crmCaseType-editable-times {
+    bottom: -24px;
+    cursor: pointer;
+    height: 24px;
+    left: 30px;
+    position: absolute;
+    text-align: center;
+    width: 30px;
+    z-index: 1;
+}
```

Comment:
I'm not sure why this is necessary. If you reuse the classes from standard crm-editable then these buttons should "just work" and when a theme like Shoreditch comes in to restyle everything it won't need to make a special extra bit of css just for these 2 buttons.

Reply:
Resolved this.

Reply:
Thanks